frontend/src/App.test.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+test('Test', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9115139)
```suggestion
test("Test", () => {
```